planner/core/task.go

```diff
@@ -1289,6 +1300,82 @@ func BuildFinalModeAggregation(
 	return
 }
 
+// convertAvgForMPP converts avg() to if(count(arg)=0, null, sum(arg)/count(arg)), in detail:
```

Comment:
add a comment: If there is no agg, return nil.

Reply:
ok